core/utils_test.py

```diff
@@ -880,6 +880,17 @@ def test_get_human_readable_time_string(self) -> None:
             AssertionError, 'Time cannot be negative'):
             utils.get_human_readable_time_string(-1.42)
 
+    def test_get_number_of_days_since_date(self) -> None:
+        self.assertEqual(
+            90,
+            utils.get_number_of_days_since_date(
+                datetime.date.today() - datetime.timedelta(days=90)
+            )
+        )
+        with self.assertRaisesRegex(
+            AssertionError, 'Time cannot be negative'):
+            utils.get_human_readable_time_string(-1.42)
```

Comment:
I think this is the wrong action under test. Shouldn't it be `utils.get_number_of_days_since_date(datetime.date.today() + datetime.timedelta(days=1))`?

Reply:
Also, both assertions should each be in a separate test. One test that handles non-negative time and the other test that handles negative time.

Reply:
> Also, both assertions should each be in a separate test. One test that handles non-negative time and the other test that handles negative time.

Ah, it is actually not important to keep `'Time cannot be negative'` assertion in this case.

Reply:
@masterboy376 Oh. Then, the entire `with` statement should be removed.

Reply:
Done

Reply:
@masterboy376 For future reference, leave the "Done" comment *after* you've pushed the relevant changes to your PR, not *before*.